web/client/epics/__tests__/timeline-test.js

```diff
@@ -202,13 +202,13 @@ describe('timeline Epics', () => {
             testEpic(syncTimelineGuideLayer, NUM_ACTIONS, [selectLayer('TEST_LAYER2'), changeLayerProperties('TEST_LAYER', {visibility: false}) ], (actions) => {
                 doAssertion(NUM_ACTIONS, actions, 'TEST_LAYER1');
                 done();
-            }, STATE_TIMELINE);
+            }, STATE_TIMELINE, done());
```

Comment:
@MV88 I keep getting a timeout without calling the `done` function. Any suggestion?

Reply:
probably is because the internal done() is never called 
did you debug that test?
you can use npm run continuoustest and then edit tests.webpack.js with something like
```js
var context = require.context('../web/client/epics', true, /(timeline-test\.jsx?)|(-test-chrome\.jsx?)$/);
context.keys().forEach(context);
module.exports = context;
```

in this way with `npm run continuoustest` you cna run epics tests in isolation and you can do quicker checks


Anyway I don't get why you need to edit a unit test for an epic if you haven't changed its code

Reply:
well noted. taking a closer look